jmx/jmx.go

```diff
@@ -63,7 +64,7 @@ func (cfg *connectionConfig) command() []string {
 	if os.Getenv("NR_JMX_TOOL") != "" {
 		c = strings.Split(os.Getenv("NR_JMX_TOOL"), " ")
```

Comment:
why do we do the split here? Will this work with spaces inside the path? On windows, we install the tool on "c:\Program Files\New Relic\nrjmx\nrjmx.bat" which has some spaces.

Reply:
it works either way, I just tested it on Mac running a nrjmx tool with spaces in the patch and within the executable name itself